crates/cxx-qt-gen/src/generator/cpp/property/mod.rs

```diff
@@ -71,18 +72,39 @@ pub fn generate_cpp_properties(
 mod tests {
     use super::*;
 
-    use crate::parser::property::QPropertyFlags;
+    use core::result::Result;
```

Comment:
instead of an import that shadows you could use the fully qualified name for the returns...